packages/plugin-core/src/test/suite-integ/OpenLink.test.ts

```diff
@@ -54,10 +52,9 @@ suite("OpenLink", function () {
     });
   });
 
-  //TODO
   test.skip("grab a URL under the cursor.", (done) => {
```

Comment:
This test is skipped because it actually opens a link. I don't imagine maintainers want to deal with an unexpected pop-up when testing the plugin.

It may be a good Idea to test this on mac/Linux before shipping (I'm on windows).

Reply:
For this case, you can use `stubs` as a way of checking for functionality without invoking side effects.
See our docs about doing this here -> https://wiki.dendron.so/notes/99q7A73uGmCwu2KvSHZro.html#stubs

You can see an example of how it is used [here](https://github.com/dendronhq/dendron/blob/master/packages/plugin-core/src/test/suite-integ/DoctorCommand.test.ts#L169:L169)